_docs/install-and-upgrade.md

```diff
@@ -0,0 +1,223 @@
+to install this package from the AUR. For example, to install it using an AUR
+helper:
+
+```
+pacaur -aS thelounge
+yaourt -aS thelounge
+```
+
+Then enable and start the `thelounge.service` unit using:
+
+```
+systemctl enable --now thelounge.service
+```
+
+The Lounge is now up and running **in private mode** at <http://localhost:9000>.
+
+The package provides both a system and a user service to allow you to run The
+Lounge as your liking. Simply build and install the AUR package, and start the
+service:
+
+- **[System](https://wiki.archlinux.org/index.php/Systemd):**
+  `systemctl start thelounge.service`. The configuration is stored in
+  `/etc/thelounge/config.js`and runs as `thelounge` user.
+- **[User](https://wiki.archlinux.org/index.php/Systemd/User):**
+  `systemctl --user start thelounge.service`. The configuration is stored in
+  your home directory in `~/.thelounge/config.js`.
+
+To configure The Lounge, go to
+[the configuration section](/docs/server/configuration.html).
+
+You might want to serve The Lounge behind a reverse proxy such as Nginx. To know
+more about the benefits and steps, follow
+[the guide on reverse proxies](/docs/guides/reverse-proxies.html).
+
+## From npm releases
+
+Installing the [npm package](https://www.npmjs.com/package/thelounge) directly
+allows you to use stable releases on systems where we do not provide native
+packages.
+
+First, make sure [Node.js](https://nodejs.org/) v6.13.0 (or more recent) and [Yarn](https://yarnpkg.com/) are installed on your system. See [official instructions for Node.js](https://nodejs.org/en/download/package-manager/) and [official instructions for Yarn](https://yarnpkg.com/en/docs/install).
+
+Then install The Lounge using:
+
+```
+yarn global add thelounge
+```
+
+{: .alert.alert-info role="alert"}
+If you install The Lounge using the [npm](https://docs.npmjs.com/cli/npm) CLI instead, you will need the `--unsafe-perm` flag: `npm install --global --unsafe-perm thelounge`
+
+Start the server manually using:
+
+```sh
+thelounge start
+```
+
+{: .alert.alert-info role="alert"}
+Note that installing from npm does not daemonize nor autostart The Lounge.
+
+The Lounge is now up and running **in private mode** at <http://localhost:9000>.
+
+Read more about how to use The Lounge from the command line in
+[the CLI usage section](/docs/server/usage.html).
+
+Its configuration file is located at `~/.thelounge/config.js`. To configure The
+Lounge, go to [the configuration section](/docs/server/configuration.html).
+
+To upgrade The Lounge, simply re-run the `install` command above, and restart it.
+
+You might want to serve The Lounge behind a reverse proxy such as Nginx. To know
+more about the benefits and steps, follow
+[the guide on reverse proxies ](/docs/guides/reverse-proxies.html).
+
+## Docker
+
+Official Docker images of The Lounge are made available for every new release on
+[DockerHub](https://hub.docker.com/r/thelounge/lounge/).
```

Comment:
This is the old DockerHub repo (thelounge is the new name)

Reply:
Great, fixed it along with the following, let me know if that's correct:

```diff
──────────────────────────────────────
modified: _docs/install-and-upgrade.md
──────────────────────────────────────
@ install-and-upgrade.md:105 @ To upgrade The Lounge, simply re-run the `install` command above, and restart it
 ## Docker

 Official Docker images of The Lounge are made available for every new release on
-[DockerHub](https://hub.docker.com/r/thelounge/lounge/).
+[DockerHub](https://hub.docker.com/r/thelounge/thelounge/).

 To run a container using [Docker Compose](https://docs.docker.com/compose/),
 use [this example `docker-compose.yml`
-file](https://github.com/thelounge/docker-lounge/blob/master/docker-compose.yml)
+file](https://github.com/thelounge/thelounge-docker/blob/master/docker-compose.yml)
 and run:

@ install-and-upgrade.md:120 @ Otherwise, run:
 docker run --detach \
-   --name lounge \
+   --name thelounge \
    --publish 9000:9000 \
-   --volume ~/.lounge:/home/lounge/data \
+   --volume ~/.thelounge:/var/opt/thelounge \
    --restart always \
-   thelounge/lounge:latest
+   thelounge/thelounge:latest

 For more details about using The Lounge with Docker, refer to
-[the official repository](https://github.com/thelounge/docker-lounge#docker-lounge).
+[the official repository](https://github.com/thelounge/thelounge-docker).
```